bokehjs/examples/anscombe/anscombe.ts

```diff
@@ -1,8 +1,8 @@
 namespace Anscombe {
   console.log(`Bokeh ${Bokeh.version}`);
-  Bokeh.set_log_level("debug");
+  Bokeh.set_log_level("info");
 
-  const anscombe_quartet = Bokeh.LinAlg.transpose([
+  const anscombe_quartet = Bokeh.Utils.transpose([
```

Comment:
There are a few more things that have to be exposed through this API, that's why I would like to defer this to another PR.